QuizApp/App/Screens/Quiz/QuizListViewModel.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+final class QuizListViewModel {
+
+    @Published var quizes: [QuizViewModel] = []
+
+    private let quizUseCase: QuizUseCaseProtocol
+
+    init(quizUseCase: QuizUseCaseProtocol) {
+        self.quizUseCase = quizUseCase
+    }
+
+}
+
+extension QuizListViewModel {
+
+    func fetchQuiz() {
+        Task(priority: .background) {
+            let quizes = try await quizUseCase.fetchQuizes()
+
+            DispatchQueue.main.async { [weak self] in
+                guard let self = self else { return }
+                _ = quizes.map { model in
```

Comment:
`map` must always return some value that is used. It would be best if you used forEach for your use case, but I recommend that you always assign values atomically. Also, what if you load quizzes multiple times? This will just append the values.
Why not
```
self.quizes = quizes
    .map { QuizViewModel(from: $0) }
```